utils.py

```diff
@@ -48,6 +48,7 @@
 CSV_ANDY = "1,some title,Articles,1,1,some subtitle,the abstract,Published,key1|key2|key3,2018-01-01T09:00:00,2018-01-02T09:00:00,10.1000/xyz123,1,3,3,Y,https://creativecommons.org/licenses/by/4.0/,Mr,Andy,James Robert,Byers,Jr,BirkbeckCTP,Andy's Bio,abyers@journal.com,N,,,,"
 
 
+plugin_loader.load()
```

Comment:
I'm slightly concerned about this, why do we have to call the loader outside of the tests?

Reply:
Good call--I had put this here before creating the mock_import_stages kwarg, so the test was failing because this variable was assigned before the plugins were loaded. I'll remove it.